src/lib/index.ts

```diff
@@ -98,7 +100,7 @@ export class GoogleToken {
     if (!this.key && this.keyFile) {
       const mimeType = mime.getType(this.keyFile);
       if (mimeType === 'application/json') {
```

Comment:
Suggestion. Isn't it better to use `switch` instead of `if-else-if` chain?

Reply:
Why not :)  Switched it up.